test/functional/artefacts_controller_test.rb

```diff
@@ -153,23 +153,13 @@ class ArtefactsControllerTest < ActionController::TestCase
         assert_select "li[class~=active] a[href=/artefacts/new]"
       end
 
-      context 'if publisher is an app for which the taging has NOT been migrated' do
-
-        should 'render the tags partial if publisher is not an app with migrated tagging' do
-          Settings.stubs(:apps_with_migrated_tagging).returns(%w(smartanswers testapp))
+      context "when publisher app has not been migrated" do
```

Comment:
To clarify this change for other reviewers - beforehand we were testing multiple potential values of the "migrated apps" list, and we already had special cases for when publisher is migrated/non migrated, even though we haven't migrated this yet.

We think it is simpler to just test the current behaviour of the app, so NON_MIGRATED_APPS is now a constant and we have removed one of the contradictory tests here.